src/index.js

```diff
@@ -0,0 +1,105 @@
+import './style.css';
+import moreIcon from './more.svg';
+import Icon from './enter.svg';
+import recycle from './recycle.svg';
+
+function elementGenerator(typeName, className, content, idName) {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.